index.html

```diff
@@ -37,23 +37,301 @@ <h6>
+                    <p>
+                        Hello I'm a soffware developer! I can help you build a product, 
+                        feature or website Look through some of my work and experience!
+                        if you like what you see and have a project you need coded, 
+                        don't hestiate to contact me.
+                    </p>
+                </div>
+                <div class="social-media-title">
+                    <h6>
+                        LET'S CONNECT
+                    </h6>
+                </div>
+                <ul class="aboutme-social-media-list">
+                    <li>
+                        <a href="url" class="aboutme-twitter-icon"><img src="./img/icon-tiwtter.svg" alt=""></a>
+                    </li>
+                    <li>
+                        <a href="url" class="aboutme-linkedin-icon"><img src="./img/icon-linkedin.svg" alt=""></a>
+                    </li>
+                    <li>
+                        <a href="url" class="aboutme-m-icon"><img src="./img/icon-m.svg" alt=""></a>
+                    </li>
+                    <li>
+                        <a href="url" class="aboutme-github-icon"><img src="./img/icon-github.svg" alt=""></a>
+                    </li>
+                    <li>
+                        <a href="url" class="aboutme-free-icon"><img src="./img/icon-free.svg" alt=""></a>
+                    </li>
+                </ul>
+            </div>
+            <button class="aboutme-secondary-btn"><p>Get my resume</p></button>
+            <ul class="skill-list">
+                <li class="languages">
+                  <div class="arrow">
+                    <h2 class="lang-heading">Languages</h2>
+                    <div class="down-arrow">
+                      <img
+                        class="down-arrow"
+                        src="./img/down-arrow.svg"
+                        alt="down arrow shap"
+                      />
+                    </div>
+                  </div>
+                  <div class="language-list">
+                    <div class="logo-wrapper">
+                      <div class="logo-lang">
+                        <img src="./img/javascript.svg" alt="JavaScript logo" />
+                      </div>
+                      <div class="text-logo"><h3>JavaScript</h3></div>
+                    </div>
+                    <div class="logo-wrapper">
+                      <div class="logo-lang">
+                        <img src="./img/html5.svg" alt="html logo" />
+                      </div>
+                      <div class="text-logo"><h3>html</h3></div>
+                    </div>
+                    <div class="logo-wrapper">
+                      <div class="logo-lang">
+                        <img src="./img/css3.svg" alt="css logo" />
+                      </div>
+                      <div class="text-logo"><h3>css</h3></div>
+                    </div>
+                  </div>
```

Comment:
- [ ] The technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul`, and each of them a `li`. Please kindly fix that 😁😉